CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # Changelog
 
+## [1.58](https://github.com/brave/brave-ios/releases/tag/v1.58)
+
+### Web3
+
+- Added Coinbase Pay on-ramp for Brave Wallet buy flows. [(#7880](https://github.com/brave/brave-ios/issues/7880))
+- Added Stripe on-ramp for Brave Wallet buy flows. ([#7794](https://github.com/brave/brave-ios/issues/7794))
+- Added Filecoin support in Brave Wallet. ([#7809](https://github.com/brave/brave-ios/issues/7809))
+- Added the ability for users to choose which Solana account to choose when connecting to Solana DApps. ([#7637](https://github.com/brave/brave-ios/issues/7637))
```

Comment:
Maybe reword to `Added ability to choose Solana account when connecting to Solana DApp.`